### ISSUE TITLE
Replication of Tanks and Temples results

### ISSUE THREAD

**Issue by pedrogcmartin** [Kai-46/nerfplusplus#55]
Hi there,

I downloaded the pre-processed data and pre-trained weights from T&T, rendered the test set for each scene, stored the PSNR outputs for each frame and calculated the average for each scene and did not get results close to what was published in your paper.

M60: 18.47 dB (paper: 17.88 dB)
Playground: 17.80 dB (paper: 22.37 dB)
Train: 22.70 dB (paper: 17.17 dB)
Truck: 22.26 dB (paper: 22.77 dB)

I would like to know if you used any different configuration to generate your results.

Thank you very much.

**Comment by wbl66** [comment]
Hello，
The dataset link now is invalid. Could you please provide it for me? 
Thank you very much！！

**Comment by Gansche** [comment]
Hello，
The dataset link now is invalid. Could you please provide it for me?
Thank you very much！！
